app/bundles/LeadBundle/Entity/ContactExportScheduler.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Entity;
+
+use DateTimeImmutable;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
+use Mautic\UserBundle\Entity\User;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Mapping\ClassMetadata as ValidatorClassMetadata;
+
+class ContactExportScheduler
+{
+    private int $id;
```

Comment:
I think this should be
```suggestion
    private ?int $id = null;
```
And then we can avoid ignoring
```
- '#Property [a-zA-Z0-9\\_]+::\$id is never written, only read.#'
```
in phpstan.neon

In fact, I'd recommend to make all entity properties nullable. The only properties that can have a strict type are those set via constructor. Otherwise accessing them like
```php
$contactExportScheduler = new ContactExportScheduler();
$contactExportScheduler->getId();
```
will end up with an error as the value of the property is not set to an int before accessing it.